### PR TITLE
Add column tag from airtable option with colors

### PR DESCRIPTION
**Pull request by SondreKanstad**
Airtable allows categorical data which is set as options. Options are null or a list of possible inputs for a cell value. These options can also have meta data about the color used to differentiate the categories in the Airtable UI. This PR creates tag for cellvalues that are options, and adds background and textcolor to options that has this metadata.

Before:
image

After:
image

### CODE REVIEW

File: frontend/beCompliant/src/components/questionRow/QuestionRow.tsx [hunk 1]
```diff
@@ -15,12 +16,12 @@ interface QuestionRowProps {
...
-  setFetchNewAnswers,
-  tableColumns,
-  team,
-}: QuestionRowProps) => {
+                              record,
+                              choices,
+                              setFetchNewAnswers,
+                              tableColumns,
+                              team,
+                            }: QuestionRowProps) => {
```
Comment (andersstandal) [inline frontend/beCompliant/src/components/questionRow/QuestionRow.tsx:24]: Jøss

File: frontend/beCompliant/src/components/questionRow/QuestionRow.tsx [hunk 2]
```diff
@@ -43,13 +44,22 @@ export const QuestionRow = ({
           );
         }
         const cellValue = record.fields[columnKey];
+
+        if(cellValue === undefined){
```
Comment (andersstandal) [inline frontend/beCompliant/src/components/questionRow/QuestionRow.tsx:48]: På slike statements så kan man som regel bare skrive `!cellValue` 💯

**Review by andersstandal — APPROVED** [review]
Hippiti hoppiti, mergity deploiti

File: frontend/beCompliant/src/components/questionRow/QuestionRow.tsx (hunk 1 above)
Comment (SondreKanstad) [reply frontend/beCompliant/src/components/questionRow/QuestionRow.tsx:24]: Autoformatert fra intellij, så vi lar det bli😵‍💫

**Pull request merged**